keps/sig-scheduling/2891-simplified-config/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Simplified Scheduler Config
+kep-number: 2891
+authors:
+  - "@damemi"
+owning-sig: sig-scheduling
+participating-sigs:
+status: provisional
+creation-date: 2021-08-25
+reviewers:
+  - "@ahg-g"
+  - "@alculquicondor"
+  - "@Huang-Wei"
+approvers:
+  - "@ahg-g"
+  - "@alculquicondor"
+  - "@Huang-Wei"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+see-also:
+  - "keps/sig-scheduling/624-scheduling-framework"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
We probably need to start as beta, as that is the API maturity that we are targeting.